mypyc/ops_misc.py

```diff
@@ -58,12 +58,61 @@
                     error_kind=ERR_NEVER,
                     emit=call_emit('PyIter_Next'))
 
+# Do a next, don't swallow StopIteration, but also don't
+# propagate an error. It *can* return NULL.
```

Comment:
Can the NULL return cause trouble with refcount handling (dereferencing it)? It this is possible, maybe add a warning here.